scss/detail.scss

```diff
@@ -118,35 +139,49 @@ $max-width-mobile: 1024px;
   }
 
   .badge {
-    margin-top: 35px;
+    margin: auto;
   
     &::after {
-      bottom: 240px;
-      height: calc($badge-size * 30%);
+      bottom: 250px;
+      height: calc(100px * 0.7);
       left: 165px;
-      width: calc($badge-size * 30%);
+      width: calc(100px * 0.7);
```

Comment:
well yes, but actually no.jpg 
In the function `calc`, this is equal always to 70px. You're going to look its magic when you use it with relative units